src/App.tsx

```diff
@@ -20,27 +20,31 @@ const App = () => {
 
         <Background>
           <BrowserRouter>
-            <Switch>
-              <Redirect exact to="/home" from="/" />
-
-              <Route exact path="/linkedin" component={ LinkedInCallback } />
-
-              <Route path="/login">
-                <Login />
-              </Route>
-
-              <Route path="/sign-up">
-                <SignUp />
-              </Route>
-
-              <PrivateRoute path="/create-profile">
-                <CreateProfile />
-              </PrivateRoute>
-
-              <PrivateRoute path="/home">
-                <Home />
-              </PrivateRoute>
-            </Switch>
+            <Routes>
+              <Route path="/" element={ <Navigate to="home" replace /> } />
+
+              <Route path="linkedin" element={ LinkedInCallback } />
+
+              <Route path="login" element={ <Login /> } />
+
+              <Route path="sign-up" element={ <SignUp /> } />
+              <Route
```

Comment:
We have to wrap PrivateRoutes in a Route component in v6, as only <Route>s can be children of <Routes>